CONTRIBUTING.md

```diff
@@ -187,6 +187,7 @@ bundle exec rake bin/rails db:migrate RAILS_ENV=test
 ### Run the specs
 ```  bash
 # update your apps gemfile to point at the local version of the gem. e.g.:
+# if you have cloned bulkrax into a folder titled "vendor" in your project
```

Comment:
@alishaevn to me, this reads kind of funny with the "e.g." at the end of the line above. I think if you put this line above 189 it would read easier

Reply:
updated.